frontend/13f-app/src/App.js

```diff
@@ -4,6 +4,9 @@ import Footer from './components/Footer'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import Home from './components/pages/home'
 import SearchResults from './components/pages/searchResults'
+import FilingsPage from './components/pages/filinglistpage'
+import DataPage from './components/pages/datapage'
```

Comment:
```suggestion
import SearchResults from './components/pages/SearchResults'
import FilingsPage from './components/pages/FilingsListPage'
import DataPage from './components/pages/DataPage'
```
This is definitely up to you, but if you're asked to submit a [work sample test](https://www.forbes.com/sites/quora/2017/02/23/work-sample-tests-should-be-the-future-of-software-engineering-interviews/?sh=2b59dcfd7dc4) this will reflect more favorably than not, assuming the rest of your work is up to snuff. (The change being suggested here will have to be reflected in the filenames as well because JS imports are case-sensitive)

[AirBnb Style Guide for General Naming Conventions](https://github.com/airbnb/javascript/tree/master/react#naming)